tests/publiccloud/prepare_tools.pm

```diff
@@ -20,6 +20,40 @@ use registration 'add_suseconnect_product';
 use version_utils qw(is_sle is_opensuse);
 use repo_tools 'generate_version';
 
+sub install_in_venv {
+    my ($pip_packages, $binary) = @_;
+    die("Missing pip packages") unless ($pip_packages);
+    die("Missing binary name")  unless ($binary);
+    $pip_packages = [$pip_packages] unless ref $pip_packages eq 'ARRAY';
+
+    my $venv = $binary;
+    $venv =~ s/[^\w]+//g;
```

Comment:
why we need this ?

Reply:
To be on the save side that the name doesn't have some special chars - sanity

Reply:
I actually added this, as I previously used the `$pip_package` for the name. If you see a problem with it.. I can remove it

Reply:
( just deleted long comment expressing my thoughts about this check ) 

yes please

Reply:
done